src/layers/convolutional.ts

```diff
@@ -365,18 +371,47 @@ export abstract class Conv extends Layer {
             `convolution, but received ${JSON.stringify(this.dilationRate)}`);
       }
     }
+  }
 
-    this.activation = getActivation(config.activation);
-    this.useBias = config.useBias == null ? true : config.useBias;
+  protected static verifyConfig(config: BaseConvLayerConfig) {
```

Comment:
Why static, this seems a little odd to me.

Reply:
Originally I had them non-static, but was bitten by the following behavior:

imagine class B inherits from class A, and both A and B have method named `verifyConfig`

A.constructor calls `this.verifyConfig(config)`
B.constructor calls `super(config)` and then `this.verifyConfig()`

when creating a new B, the value of `this` is of type B, even when referenced by A's constructor. 
 Thus, the `verifyConfig` defined in B is called twice and verifyConfig defined in `A` is not called at all.

Making these calls static replaces calls to `this.verifyConfig` with `A.verifyConfig`.   Here is the reference I used to get a handle on this:
https://javascript.info/class-inheritance